diff --git addons/auth_oauth/models/res_users.py addons/auth_oauth/models/res_users.py
index f48802c..b10f19a 100644
--- addons/auth_oauth/models/res_users.py
+++ addons/auth_oauth/models/res_users.py
@@ -10,6 +10,8 @@ from odoo import api, fields, models
 from odoo.exceptions import AccessDenied
 from odoo.addons.auth_signup.models.res_users import SignupError
 
+from odoo.addons import base
+base.res.res_users.USER_PRIVATE_FIELDS.append('oauth_access_token')
 
 class ResUsers(models.Model):
     _inherit = 'res.users'
