diff --git addons/anonymization/wizard/anonymize_wizard.py addons/anonymization/wizard/anonymize_wizard.py
index a0b355d..e8901d3 100644
--- addons/anonymization/wizard/anonymize_wizard.py
+++ addons/anonymization/wizard/anonymize_wizard.py
@@ -4,10 +4,7 @@
 import base64
 import os
 import random
-try:
-    import cPickle as pickle
-except ImportError:
-    import pickle
+
 from lxml import etree
 from operator import itemgetter
 
@@ -15,6 +12,7 @@ from odoo import api, fields, models, _
 from odoo.exceptions import UserError
 from odoo.release import version_info
+from odoo.tools import pickle
 from odoo.tools.safe_eval import safe_eval
 from odoo.addons.anonymization.models.anonymization import group
 
 WIZARD_ANONYMIZATION_STATES = [('clear', 'Clear'), ('anonymized', 'Anonymized'), ('unstable', 'Unstable')]
@@ -207,7 +205,7 @@ class IrModelFieldsAnonymizeWizard(models.TransientModel):
 
         # save pickle:
         fn = open(abs_filepath, 'w')
-        pickle.dump(data, fn, pickle.HIGHEST_PROTOCOL)
+        pickle.dump(data, fn, protocol=-1)
 
         # update the anonymization fields:
         ano_fields.write({'state': 'anonymized'})
